Sources/StreamChatUI/Utils/ChatChannelNamer_Tests.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  ChatChannelNamer_Tests.swift
+//  StreamChatUI
+//
+//  Created by Dominik Bucher on 01.03.2021.
+//  Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
```

Comment:
How come the formatter missed this?

Reply:
I fixed this manually, but I have same question 😅